### PR TITLE
Whitepages

### PR DESCRIPTION
**Pull request by scortiiz**
Completed infopage + infopopup, still need to design the adminpopup and add a edit and remove bubble function

### CODE REVIEW

File: src/app/components/whitepagepopup/adminpopup/addpopup/addpopup.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.popupOverlay {
+    position: fixed;
+    top: 0;
```
Comment (brandonwu32) [inline src/app/components/whitepagepopup/adminpopup/addpopup/addpopup.css:3]: I'm not sure if the popup has been finished yet, if it has, can you let me know how I can see it?

File: src/app/components/whitepagepopup/infopopup/infopopup.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
+.popupOverlay {
+    position: fixed;
+    top: 0;
+    left: 0;
```
Comment (brandonwu32) [inline src/app/components/whitepagepopup/infopopup/infopopup.css:4]: I don't know how to view this one too

File: src/app/page.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import Image from "next/image";
 import styles from "./page.module.css";
 
-import Navbar from "./components/navbar/Navbar";
-
+import Navbar from "./components/navbar/navbar";
```
Comment (brandonwu32) [inline src/app/page.js:4]: remove comment

File: src/app/whitepages/adminview/editview/page.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  const togglePopUp = () => {
+    setButtonPopUp(!buttonPopUp);
+  }
+
+  return (
+    <div className={styles.page}>
+      <div className='top'>
+        <p className = 'title'>Team</p>
+        <div className = 'btz'>
+          <button className="red" id="large">Edititing</button>
```
Comment (brandonwu32) [inline src/app/whitepages/adminview/editview/page.js:28]: Buttons need linking functionality

File: src/app/whitepages/adminview/page.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
+'use client'
+import Image from "next/image";
+import styles from "./page.css";
+import { useState } from "react";
```
Comment (brandonwu32) [inline src/app/whitepages/adminview/page.js:4]: The circles on adminview are a different size than the circles on other pages

**Review by brandonwu32 — COMMENTED** [review]
Great job using react hooks to account for the add and remove functionality that we still need to implement. We can give you guys a little more time to finish this task since there are a lot of moving parts. Something to look at is the figma mockup to see if the things on the page line up.

**Comment by scortiiz** [comment]
> Great job using react hooks to account for the add and remove functionality that we still need to implement. We can give you guys a little more time to finish this task since there are a lot of moving parts. Something to look at is the figma mockup to see if the things on the page line up.
I was working with tanya for a while trying to figure out how to add/remove and we ended up realizing it's only possible when we implement back end.

**Comment by tbhakhri** [comment]
> > I was working with tanya for a while trying to figure out how to add/remove and we ended up realizing it's only possible when we implement back end.

@scortiiz yup that's correct! just make sure popups are being triggered, and if possible try to console.log the information in each text box since it'll make people's lives easier when we connect this to backend

File: src/app/components/button/button.js [hunk 6]
```diff
@@ -9,12 +9,11 @@ export default function Button(props) {
         }
     };
 
-
     return (
         <div className="button">
```
Comment (tbhakhri) [inline src/app/components/button/button.js:13]: looks like you're modifying the button component and are taking away props.color and props.size  and are putting in props.image instead. this will cause problems for people using this component in the way it initially was. can you revert these changes in this file, and make a new file for an image button instead?

File: src/app/components/whitepagebubble/whitepagebubble.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    width: 100%;               
+    height: 100%;              
+    object-fit: cover;         /* Ensures the image covers the area */     
+    border: 20px solid #02A0D4;
+    border-radius: 30%;
+}
+.popup-bubble {
+    width: 50%;       /* Set a smaller width */
+    height: 70%;      /* Set a smaller height */
+    border: 5px solid #02A0D4; /* Adjust border size if needed */
```
Comment (tbhakhri) [inline src/app/components/whitepagebubble/whitepagebubble.css:26]: percentages here too

File: src/app/components/whitepagebubble/whitepagebubble.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import './whitepagebubble.css'
+import Image from "next/image";
+
+
+export default function WhitePageBubble(props) {
+    function handleClick() {
+        if (props.onClick) {
+            props.onClick(); 
+        }
+        console.log("yee");
```
Comment (tbhakhri) [inline src/app/components/whitepagebubble/whitepagebubble.js:10]: this was added for testing purposes, prob remove this console.log now

**Review by tbhakhri — CHANGES_REQUESTED** [review]
Overall great job! Address all comments, but two main things to fix: 1. you want to use percentages rather than pixels 2. loop rather than copying code multiple times. 

when coding, you always want to think ahead -- in practice, we might not know how many people seacology will have in their organization (just as an example), or other things as well. when we do front-end, we aren't just  trying to making stuff just "look" functional. we want to have things working in such a way that it's easy to then link to the backend. go through your code and think about what we want members of seacology to be able to do with whitepages. does your code make it easy to do what you want? does your code account for variable numbers of members _on every page?_  is there anything else you can think of functionality wise that might need to be adapted?

 i'm leaving that open-ended since i didn't comment on everything because i want you guys to think about it on your own, but would like to see some progress in your next PR!

File: src/app/components/whitepagepopup/adminpopup/addpopup/addpopup.css (hunk 1 above)
Comment (scortiiz) [reply src/app/components/whitepagepopup/adminpopup/addpopup/addpopup.css:3]: I made it so now when you click the add member that one shows and then with the bubble the edit member one shows

File: src/app/page.js (hunk 3 above)
Comment (scortiiz) [reply src/app/page.js:4]: donee

**Review by tbhakhri — COMMENTED** [review]
Great progress! Last part of the feedback still remaining (convert all pixels to percentages except border radius, and there's one more spot where looping should be implemented) but you're almost there!
